### PR TITLE
Fix Czech localization

### PR DESCRIPTION
**Pull request by R3gi**
Fix typos

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
@R3gi Thanks for contribution. Do you know any native speaker who can review these changes?

**Comment by R3gi** [comment]
Unfortunately not. But those improvements are generally just small typographical changes to punctuation etc. They don't affect meaning at all, so it should be safe to merge.

Maybe one of the past contributors could do the code review? I'll eventually ping them once I'll be on desktop…

**Comment by alexandrtovmach** [comment]
@R3gi Don't give up 😄 You can ask your friends, or find reviewers on some local developer forums. I actually did in the same way for Ukrainian and Russian translations. Probably it works for you

File: lang/cs/index.md [hunk 1]
```diff
@@ -13,18 +13,18 @@ Shrnutí
...
+2. MINOR – když se přidá funkcionalita se zachováním zpětné kompatibility
 3. PATCH - když se opravila chyba a zůstala kompatibilita
 
 Pomocí předběžných verzí a přidáváním metadat je možné upřesnit informace. Např.: `1.0.0-alfa`, `1.0.1-beta+2`
 
 Úvod
 ----
 
-Ve světě softwaru, jeho vývoje a správy existuje děsivé místo, kterému anglicky říkáme “dependency hell” (v překladu "peklo závislostí"). Čím větší a komplexnější váš projekt je, tím je větší šance, že se na tomto místě jednou ocitnete.
+Ve světě softwaru, jeho vývoje a správy existuje děsivé místo, kterému anglicky říkáme „dependency hell“ (v překladu „peklo závislostí“). Čím větší a komplexnější váš projekt je, tím je větší šance, že se na tomto místě jednou ocitnete.
```
Comment (alexandrtovmach) [inline lang/cs/index.md:25]:
```suggestion
Ve světě softwaru, jeho vývoje a správy existuje děsivé místo, kterému anglicky říkáme "dependency hell" (v překladu "peklo závislostí"). Čím větší a komplexnější váš projekt je, tím je větší šance, že se na tomto místě jednou ocitnete.
```

Comment (alexandrtovmach) [reply lang/cs/index.md:25]: Could you use default doublequote `"` everywhere, to make it consistent? Thanks

**Review by alexandrtovmach — COMMENTED** [review]
It's not standardized yet, but could you replace all quotes with `"`. I'm going to investigate in linting and formatting process in near future to avoid this

File: lang/cs/index.md (hunk 1 above)
Comment (R3gi) [reply lang/cs/index.md:25]: Sure, but it's incorrect to use English quotation marks (") here. In Czech we are using „“ instead. That's why I changed it.

Does it mean that you can somehow automagically replace default English marks with correct quotation marks (depending on various languages)? I'm not sure if this is a good approach.

**Comment by R3gi** [comment]
@orangesunny Could you please review my changes 😊

File: lang/cs/index.md (hunk 1 above)
Comment (alexandrtovmach) [reply lang/cs/index.md:25]: Ohhh.. So it's locale preferences, I didn't know. Learned something new this evening 💪 
I'll investigate in it to make some opinion, and will back with feedback

**Comment by alexandrtovmach** [comment]
@orangesunny Friendly reminder

**Review by orangesunny — APPROVED** [review]

**Review by orangesunny — APPROVED** [review]

**Comment by orangesunny** [comment]
> @orangesunny Friendly reminder

Hi, sorry, I don’t even know this project, and a notification got lost in my inbox. But changes seem right. :)

**Comment by alexandrtovmach** [comment]
@R3gi Could you fix conflicts?

**Comment by R3gi** [comment]
@alexandrtovmach Hi, the conflict is resolved so it should be okay now.

(Hopefully the merge of this tiny fix won't take another year… 😉)

**Pull request merged**